src/routes/search.js

```diff
@@ -99,14 +98,18 @@ class SearchRoute extends Component {
       shouldFocusItem = this.props.match.params.id || null;
     }
 
-    // always update the results state
+    // DO NOT always update the results for searchString state
```

Comment:
Could we elaborate on this comment as to why we shouldn't update the results in this case?